cmd/plasmacli/subcmd/query/info.go

```diff
@@ -43,20 +43,11 @@ var infoCmd = &cobra.Command{
 			fmt.Printf("UTXO %d\n", i)
 			fmt.Printf("Position: %s, Amount: %s, Spent: %t\nSpender Hash: %s\n", utxo.Position, utxo.Output.Amount.String(), utxo.Spent, utxo.SpenderTx)
 			fmt.Printf("Transaction Hash: 0x%x\nConfirmationHash: 0x%x\n", utxo.TxHash, utxo.ConfirmationHash)
-			/*
-				// TODO: Add --verbose flag that if set will query TxInput and print InputAddresses and InputPositions as well
-				// print inputs if applicable
-				positions := utxo.InputPositions
```

Comment:
not a field of utxo

Reply:
I think the pseudo-code was incorrect. I think the idea is to query for the input and do utxo.Position, the pseudo-code is probably from when the utxo did have `InputPositions` as a field

Reply:
Ah Gotcha. I'll update it